rules/python/lang/reflection_using_user_input.yml

```diff
@@ -0,0 +1,51 @@
+imports:
+  - python_shared_common_user_input
+  - python_shared_lang_import1
+patterns:
+  - pattern: getattr($<_>, $<USER_INPUT>$<...>)
+    filters:
+      - variable: USER_INPUT
+        detection: python_shared_common_user_input
+        scope: result
+  - pattern: globals()[$<USER_INPUT>]$<...>
```

Comment:
A bit of a weird one: `globals()[$<...>]$<...>` does not catch the example code either, but the literal `globals()[operation]$<...>` does catch. Will have to investigate further

Reply:
^ Possible fix: https://github.com/Bearer/bearer/pull/1601